packages/xl-odt-exporter/src/odt/defaultSchema/blocks.tsx

```diff
@@ -212,18 +224,60 @@ export const odtBlockMappingForDefaultSchema: BlockMapping<
 
   checkListItem: (block, exporter) => (
     <TextP>
-      <TextSpan>{block.props.checked ? "☒" : "☐"}</TextSpan>
+      <TextSpan>{block.props.checked ? "☒" : "☐"} </TextSpan>
       {exporter.transformInlineContent(block.content)}
     </TextP>
   ),
 
+  pageBreak: async () => {
+    return <TextP text:style-name="PageBreak" />;
+  },
+
   image: async (block, exporter) => {
-    await exporter.resolveFile(block.props.url);
-    return (
-      <DrawFrame>
-        <DrawImage href={`Pictures/${block.props.url}`} />
-      </DrawFrame>
+    const { Buffer } = await import("buffer");
+    const blob = await exporter.resolveFile(block.props.url);
+    const arrayBuffer = await blob.arrayBuffer();
+    const base64 = Buffer.from(arrayBuffer).toString("base64");
+    //const dimensions = await getImageDimensions(blob);
+    const styleName = createParagraphStyle(
+      exporter as ODTExporter<any, any, any>,
+      block.props
     );
+    const editorWidth = 623;
```

Comment:
what kind of unit is this / where does this number come from?

Reply:
Width of the BN editor in `px`.
I needed to somehow get a `%` value from `previewWidth`, which seems to be a `px` value.
Not sure how else to get it, other than by using `editorWidth` - let me know if you have any suggestions.

Reply:
why do you need the editorWidth?

Reply:
Manage to remove the dependency on `editorWidth`